plugins/generator-1.19.2/forge-1.19.2/templates/livingentity/livingentity.java.ftl

```diff
@@ -679,6 +684,21 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 		}
     </#if>
 
+	<#if data.boundingBoxScale??>
+	@Override public EntityDimensions getDimensions(Pose p_33597_) {

```

Comment:
Obfuscated names like `p_33597_` should be replaced with something readable, for example `pose`